edward/util/random_variables.py

```diff
@@ -778,3 +778,66 @@ def transform(x, *args, **kwargs):
   new_x = TransformedDistribution(x, bij, *args, **kwargs)
   new_x.support = new_support
   return new_x
+
+
+def marginal(x, n):
```

Comment:
Functions should be placed according to alphabetical ordering of function names.